packages/web/package.json

```diff
@@ -21,6 +21,7 @@
     "@upswyng/common": "0.1.0",
     "algoliasearch": "^3.33.0",
     "axios": "^0.19.0",
+    "cross-env": "^7.0.2",
```

Comment:
Couple things on this, it looks like this lives under `dependencies` but this should be a dev dependency since we don't want this to be bundled with the app itself. You can remove this line from the `package.json` and then:

`yarn add -D cross-env`

The `-D` will add it to dev dependencies in the `package.json` and also update the `yarn.lock` that will be part of this PR.